### ISSUE TITLE
API Sender - Providence

### ISSUE THREAD

**Issue by sarahnw** [CDCgov/prime-reportstream#6553]
This is the onboarding epic for Providence.

Their first test files can be found here https://cdc.sharepoint.com/teams/USDSatCDC/Shared%20Documents/Forms/AllItems.aspx?csf=1&web=1&e=QNMVQs&cid=ae820669%2D09a0%2D4538%2Dbaa8%2Ddfcb7857a938&FolderCTID=0x0120004368ADED9C3BEA4B94E57C5781F119FF&id=%2Fteams%2FUSDSatCDC%2FShared%20Documents%2FPRIME%20ReportStream%2FOnboarding%20%26%20Ops%2FAPI%20Senders%2FProvidence&viewid=17db3ff6%2D95c5%2D4562%2Da1f3%2D12b5097b5947

When someone picks this up, please add them to the existing email thread.

**Comment by BerniXiongA6** [comment]
@jbiskie will help handoff this epic to the engineer @GilmoreA6 who will take over cc: @BerniXiongA6

**Comment by jbiskie** [comment]
This is currently blocked by #6733.

On hold until mid-late November when Providence can staff the project.

**Comment by LaFreda1** [comment]
Closing due to no interest in COVID...will direct to full ELR
